data/types/settings/pump/suspend_threshold.go

```diff
@@ -0,0 +1,37 @@
+package pump
+
+import (
+	"github.com/tidepool-org/platform/data"
+	dataBloodGlucose "github.com/tidepool-org/platform/data/blood/glucose"
+	"github.com/tidepool-org/platform/structure"
+)
+
+type SuspendThreshold struct {
```

Comment:
Replace with `glucose.Glucose`?

Reply:
Well - I did replace with glucose.Glucose.  Not as straightforwards as the glucose target pre meal.  Had to create expose the glucose object in the framework and then add a glucose object in spotlight that is really just a reference to the Blood Object